fabtests/prov/efa/src/efa_src_addr_test.c

```diff
@@ -0,0 +1,267 @@
+/*
```

Comment:
See the documentation in the code:

```
The main difference between this program and fi_rdm_multi_client is as follows:
 * 1. This program makes the server reset the oob when a client leaves, which means
 * the server will wait for the subsequent clients to connect rather than re-using
 * the oob channel created earlier. This allows a persistent server process on one 
 * node, and multiple client process created in sequence on different nodes.
 * 2. This program will check the remote source address returned by fi_cq_readfrom
 * call on the receiver side.
```

Reply:
Maybe I should consider making `fi_rdm_multi_client` more flexible so I could just reuse it.

Reply:
Reuse `fi_rdm` in latest revision